diff --git a/dist/exports/sqlite.js b/dist/exports/sqlite.js
index 0a5ec7d8d01bab6ee96dbc2f1b4ec08336d68bfd..2f88a81d84a1be669126b82a2185e6687e7aa9de 100644
--- a/dist/exports/sqlite.js
+++ b/dist/exports/sqlite.js
@@ -1,3 +1,4 @@
+import { createRequire } from 'module';
 export { columnToCodeConverter } from '../sqlite/columnToCodeConverter.js';
 export { countDistinct } from '../sqlite/countDistinct.js';
 export { convertPathToJSONTraversal } from '../sqlite/createJSONQuery/convertPathToJSONTraversal.js';
@@ -8,7 +9,19 @@ export { dropDatabase } from '../sqlite/dropDatabase.js';
 export { execute } from '../sqlite/execute.js';
 export { init } from '../sqlite/init.js';
 export { insert } from '../sqlite/insert.js';
-export { requireDrizzleKit } from '../sqlite/requireDrizzleKit.js';
 export * from '../sqlite/types.js';
+const require = createRequire(import.meta.url);
+export const requireDrizzleKit = ()=>{
+    const moduleId = [
+        'drizzle-kit',
+        'api'
+    ].join('/');
+    const { generateSQLiteDrizzleJson, generateSQLiteMigration, pushSQLiteSchema } = require(moduleId);
+    return {
+        generateDrizzleJson: generateSQLiteDrizzleJson,
+        generateMigration: generateSQLiteMigration,
+        pushSchema: pushSQLiteSchema
+    };
+};
 
 //# sourceMappingURL=sqlite.js.map
diff --git a/dist/sqlite/requireDrizzleKit.js b/dist/sqlite/requireDrizzleKit.js
index e2ede3c94a6138c8746f018af01c14cfefbbd920..66d269e7bea6b8d78e9220e798373cd7d972be84 100644
--- a/dist/sqlite/requireDrizzleKit.js
+++ b/dist/sqlite/requireDrizzleKit.js
@@ -1,7 +1,11 @@
 import { createRequire } from 'module';
 const require = createRequire(import.meta.url);
 export const requireDrizzleKit = ()=>{
-    const { generateSQLiteDrizzleJson, generateSQLiteMigration, pushSQLiteSchema } = require('drizzle-kit/api');
+    const moduleId = [
+        'drizzle-kit',
+        'api'
+    ].join('/');
+    const { generateSQLiteDrizzleJson, generateSQLiteMigration, pushSQLiteSchema } = require(moduleId);
     return {
         generateDrizzleJson: generateSQLiteDrizzleJson,
         generateMigration: generateSQLiteMigration,
